src/app/components/showcase.tsx

```diff
@@ -0,0 +1,15 @@
+export default function Showcase() {
+  return (
+    <div className="p-5 mt-[100vh] bg-black">
+      <h1 className="pb-3 text-center font-serif text-2xl">{"Photography"}</h1>
+      <h2 className="pb-3 font-serif text-xl">{"Sports"}</h2>
+      <h2 className="pb-3 font-serif text-xl">{"Automotive"}</h2>
+      <h2 className="pb-3 font-serif text-xl">{"Landscapes"}</h2>
```

Comment:
You could probably get away with removing the `{"` since the text is just dynamic. In react, you use `{` for variables, and in this case you're basically saying "Inside this `<h2>` tag is a variable, and that variable is a string", and instead you should be able to just do:
```suggestion
      <h2 className="pb-3 font-serif text-xl">Landscapes</h2>
```

Not a huge deal either way, just cleans the code up a bit.

Reply:
Ended up cleaning this up while creating and adding the <ShowcaseItem> components